### PR TITLE
DR: use etcd DB location as a source for snapshot restore

### PR DESCRIPTION
**Pull request by vrutkovs**
Fix a typo in quorum loss scenario - it was using assets dir without a backup. `/var/lib/etcd/` should be used instead

### CODE REVIEW

File: ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ tests:
     cluster_profile: aws-centos-40
 - as: e2e-etcd-quorum-loss
   commands: |
-    recover-from-etcd-quorum-loss
+    recover-from-etcd-quorum-loss  
     TEST_SUITE=openshift/conformance/parallel run-tests
```
Comment (hexfusion) [inline ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml]: trailing whitespace?

Comment (vrutkovs) [reply ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml]: I need it to trigger the test - this commit would be removed later

Comment (hexfusion) [reply ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml]: OK gotcha I find this a nice little trick `git commit --amend --no-edit` :)

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 2]
```diff
@@ -355,7 +355,7 @@ objects:
           oc --request-timeout=5s get nodes && exit 1
 
           echo "Restore single etcd - /usr/local/bin/etcd-snapshot-restore.sh"
-          bastion_ssh core@${SURVIVING_MASTER_NODE} 'sudo -i /bin/bash -x /usr/local/bin/etcd-snapshot-restore.sh /root/assets/backup/etcd/member/snap/db $(cat /tmp/etcd_connstring)'
+          bastion_ssh core@${SURVIVING_MASTER_NODE} 'sudo -i /bin/bash -x /usr/local/bin/etcd-snapshot-restore.sh /var/lib/etcd/member/snap/db $(cat /tmp/etcd_connstring)'
```
Comment (hexfusion) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: I think `./assets/backup/etcd` is actually right. https://github.com/openshift/machine-config-operator/pull/804#issuecomment-499296463

Comment (vrutkovs) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: /close

**Pull request closed without merging**